### PR TITLE
Ms5611 posix

### PR DESCRIPTION
**Pull request by tumbili**

### CODE REVIEW

File: src/drivers/ms5611/ms5611_posix.cpp [hunk 1]
```diff
@@ -976,6 +976,7 @@ struct ms5611_bus_option &find_bus(enum MS5611_BUS busid)
 	}	
 	// FIXME - This is fatal to all threads
 	errx(1, "bus %u not started", (unsigned)busid);
+	return bus_options[0];
```
Comment (tumbili) [inline src/drivers/ms5611/ms5611_posix.cpp]: @LorenzMeier Is this the right way to make the compiler stop complaining about "reached end of non volatile function"?

Comment (LorenzMeier) [inline src/drivers/ms5611/ms5611_posix.cpp]: I recommend to change errx() to PX4_WARN(), but the missing return was indeed missing. Also make sure that the return value is still interpreted correctly for error / non-error cases.

**Comment by LorenzMeier** [comment]
#2708 looks a bit cleaner and complete - which one should we merge? @sjwilks Is #2708 already tested on hardware?

**Comment by SimonWilks** [comment]
@LorenzMeier, @tumbili and I just synced before and we will just go with #2708 

I'll rebase against master and setup a new PR but I'll test again on the Navio as a wrong raspian setup on my side might have prevented the baro working properly (which means it might already be working:).

**Pull request closed without merging**